plugin/quic/policy.go

```diff
@@ -0,0 +1,68 @@
+package quic
```

Comment:
This policy selection code has been repeated across several plugins now (`forward`, `grpc`, `quic`). Should we unify it? If so, where should it be? `plugin/pkg`?

Reply:
That would be a good place. Thanks!